### ISSUE TITLE
smurff-mm.py testing: TypeError: smurff() got an unexpected keyword argument 'side'

### ISSUE THREAD

**Issue by chupvl** [ExaScience/smurff#101]
I have an issue with the current smurff-mm.py test. 

```
conda create -n smurff
conda install -n smurff -c vanderaa smurff matrix_io -y
wget https://raw.githubusercontent.com/ExaScience/smurff/master/python/smurff/examples/smurff-mm.py
wget https://raw.githubusercontent.com/ExaScience/smurff/master/python/smurff/examples/chembl-IC50-346targets.mm
wget https://raw.githubusercontent.com/ExaScience/smurff/master/python/smurff/examples/chembl-IC50-test.mm
python smurff-mm.py
```

and the error is 

Traceback (most recent call last):
  File "smurff-mm.py", line 9, in <module>
    result = smurff(train, test, side=[('normal', []), ('normal', [])], num_latent = 1, burnin=1, nsamples=5)
  File "smurff.pyx", line 266, in smurff.smurff
TypeError: smurff() got an unexpected keyword argument 'side'

**Comment by tvandera** [comment]
Fixed in ab62d2da96dbdb1aa0974fb5aa8e7f4ffc1faba2
Maybe also install these files when doing `python setup.py install`

**Comment by chupvl** [comment]
Working perfectly well with completely new install.
